### DISCUSSION TITLE
Waiting Points

### DISCUSSION THREAD

**Discussion by RobertWW013** [Courseplay/courseplay#7412] [General]
When I save a course in CoursePlay for FS19, the vehicle goes to the start point and gives me the message that it is waiting, but I don't see anything that signifies what it is waiting for, nor do I see any way to tell it to continue. I'm very new to CoursePlay, so any help will be appreciated.

Thank you

**Comment by nitroSkirata** [comment]
Hey Robert, could you please tell us, what mode you are using, what you are trying to do and what your course looks like?

Best would be a few screenshots and/or a small vid as well

**Reply by RobertWW013** [reply]
I'm not sure how to take a screen shot, but I save a course for, say, Field 20 in Felsbrunn. I open the course via the HUD and click on "drive course". The driver goes to the first waypoint, and a blue message pops up over my minimap that says, "Driver is at the waiting point." It doesn't matter if I am coming from the garage or sitting right next to the field, it happens every time.

**Reply by nitroSkirata** [reply]
ok, you can take a screenshot by pressing the "Print Screen" Button on the keyboard, the screenshot will be saved under Documents\My Games\FarmingSimulator2019\screenshots

**Reply by RobertWW013** [reply]
[log.txt](https://github.com/Courseplay/courseplay/files/6949892/log.txt)

**Reply by RobertWW013** [reply]
I hope that file helps. It's the only one with the correct time-stamp

**Reply by RobertWW013** [reply]
I found it, @JDTiger.

**Reply by RobertWW013** [reply]
@JDTiger, I wasn't able to make that video you asked for, something about encoding issues, but I did some more research and figured out what the issue was. Thanks to your question, I was able to see that I wasn't canceling each course (I thought they self cancelled, oh, well), and that the program was waiting for the first course to finish before continuing. Understanding that, I haven't had as many issues, and the ones I do have, I've been able to work around them. Thanks again for your responese.

**Comment by Tensuko** [comment]
Our webside is not 100% complete, but most stuff is discribed there:
http://courseplay.github.io/courseplay/en/index.html
Also check YouTube for several Tutorial Videos.
Any more question, feel free to ask us.

**Reply by RobertWW013** [reply]
Thank you, @Tensuko, I have looked at that website, but the manual is a bit awkward for me to use. I haven't been able to really find what I wanted to know. I guess I just need to take some time and just peruse the website.

**Reply by RobertWW013** [reply]
@Tensuko, I have gone through the online manual a couple of times, and it is hard for me to navigate easily. Have you thought about converting it to a .pdf and just updating that as needed? I believe it would be a lot easier to navigate that way. Anyway, I have worked through most of the problems I have had, and will continue to check the website for updates.  Thank you for your response.

**Reply by Tensuko** [reply]
Won't change into a .pdf... that doesn't make any sence to scroll down a whole document...
